Polish before review (#16)
microsoft/did-x509: specification.md

2. A certificate authority fingerprint algorithm and value.
3. One or more predicates that match fields in the leaf certificate.

> The DID subject is the logical identity selected by the CA fingerprint and predicates. It is not necessarily the X.509 subject name; `subject` is only one predicate type.

Reviewer: ```suggestion
The DID subject is the logical identity selected by the CA fingerprint and sequence of predicates. It is not necessarily the X.509 subject name; `subject` is only one predicate type.
```

2. Check whether the list of certificates forms a valid certificate chain using [RFC 5280 certification path validation](https://www.rfc-editor.org/rfc/rfc5280#section-6) procedures with the last certificate in the chain as trust anchor. Implementations MUST perform RFC 5280 certification path validation, except that they MUST treat the `fulcio_issuer` extension as recognized for purposes of critical-extension processing. Additionally, fail if any certificate in the chain contains a critical extension that is neither (a) one of the extensions represented in the JSON model (`eku`, `san`, `fulcio_issuer`), nor (b) one of the following standard RFC 5280 extensions: `basicConstraints`, `keyUsage`, `nameConstraints`, `policyConstraints`, `policyMappings`, `certificatePolicies`, `inhibitAnyPolicy`.

> The path-validation time used for certificate validity-period checks is application-specific. Applications may validate the chain at resolution time, or, when resolving a did:x509 identifier for signed content, at an authenticated signing time instead. For example, this time might come from a CWT `iat` claim ([RFC 8392](https://www.rfc-editor.org/rfc/rfc8392)) or JWT `iat` claim ([RFC 7519](https://www.rfc-editor.org/rfc/rfc7519)); such a claim is not trusted time by itself and needs to be integrity protected and accepted by application policy.

Reviewer: Instead of using the current time as specified in 5280 when validating the chain, applications may choose a context-relevant point in time. For example, applications handling signed documents may choose to...

> The path-validation time used for certificate validity-period checks is application-specific. Applications may validate the chain at resolution time, or, when resolving a did:x509 identifier for signed content, at an authenticated signing time instead. For example, this time might come from a CWT `iat` claim ([RFC 8392](https://www.rfc-editor.org/rfc/rfc8392)) or JWT `iat` claim ([RFC 7519](https://www.rfc-editor.org/rfc/rfc7519)); such a claim is not trusted time by itself and needs to be integrity protected and accepted by application policy.

Reviewer: ```suggestion
Instead of using the current time as specified in [Section 6.3.1 of RFC 5280](https://www.rfc-editor.org/info/rfc5280/#section-6.1.3) when validating the chain, applications may choose a context-relevant point in time. For example, applications handling signed documents may choose to use the signing time instead, which might come from a CWT `iat` claim ([RFC 8392](https://www.rfc-editor.org/rfc/rfc8392)) or JWT `iat` claim ([RFC 7519](https://www.rfc-editor.org/rfc/rfc7519)). Such a claim is not trusted time by itself and needs to be integrity protected and accepted by application policy.
```

### X.509 trust stores

Resolution validates the supplied certificate chain with the last certificate in `x509chain` as the path-validation trust anchor. This checks the chain and DID predicates, but does not decide whether the CA or DID is acceptable to a relying party.

Relying parties make that trust decision by policy, for example through a CA trust store, an allowlist of DIDs or CA fingerprints, or other application-specific rules.

### Use of identifier contents

> While it is acceptable to use a did:x509 identifier as an opaque handle according to relying-party policy, implementers MUST NOT parse or interpret individual components of the identifier string for authorization decisions unless the identifier has been resolved against a verified certificate chain.

Reviewer: ```suggestion
While it is acceptable to use a did:x509 identifier as an opaque handle to implement a relying-party policy, implementers MUST NOT parse or interpret individual components of the identifier string for authorization decisions unless the identifier has been resolved against a verified certificate chain.
```
